DataFormats/Scouting/test/test_readRun3Scouting_cfg.py

```diff
@@ -9,10 +9,17 @@
 parser.add_argument("--vertexVersion", type=int, help="photon data format version (default: 4)", default=4)
 parser.add_argument("--inputFile", type=str, help="Input file name (default: testRun3Scouting.root)", default="testRun3Scouting.root")
 parser.add_argument("--outputFileName", type=str, help="Output file name (default: testRun3Scouting2.root)", default="testRun3Scouting2.root")
+parser.add_argument("-f", "--fixStreamerInfo", action="store_true")
 args = parser.parse_args()
 
 process = cms.Process("READ")
 
+if args.fixStreamerInfo:
+    process.FixMissingStreamerInfos = cms.Service("FixMissingStreamerInfos",
+        fileInPath = cms.untracked.FileInPath("IOPool/Input/data/fileContainingStreamerInfos_13_0_0.root")
+    )
```

Comment:
I'd prefer
```suggestion
    from IOPool.Input.fixReading_12_4_X_Files import fixReading_12_4_X_Files
    process = fixReading_12_4_X_Files(process)
```
since this would be the pattern recommended for production use cases, and I think it would be good to demonstrate that pattern here as well.

Reply:
@makortel  Is there a reason to say 
```
process = fixReading_12_4_X_Files(process)
```
instead of just
```
fixReading_12_4_X_Files(process)
```
? Just wondering why. I can see it probably works.

Reply:
> Is there a reason to say
> 
> ```
> process = fixReading_12_4_X_Files(process)
> ```
> 
> instead of just
> 
> ```
> fixReading_12_4_X_Files(process)
> ```
> 
> ? Just wondering why. I can see it probably works.

Technically both should indeed work. The first is just the pattern the `cmsDriver.py` uses for customization functions (that, for good or ill, allows returning a different `Process` object than the argument).

Reply:
Done. Tests the function also now so that is better.